chore: post about the ambassador program (#1379)
asyncapi/website: pages/blog/asyncapi-ambassador-program.md

---
title: "AsyncAPI Ambasador Program"
date: 2023-03-10T06:00:00+01:00
type: Communication
tags:
  - Ambassador Program
cover: /img/posts/ambassador-program/cover.webp
authors:
  - name: Barbaño González
    photo: /img/avatars/barbano.webp
    link: https://www.linkedin.com/in/barbano-gonzalez-moreno
excerpt: Information about AsyncAPI Ambassador Program.
featured: true
---

> AsyncAPI and its community have continued growing exponentially during the last months. As a feedback exercise, both the project and its community are getting stronger one with another. The community is the driving force that leads the initiative. Consequently, the bigger it gets, the more consistency and potential the initiative has. It’s amazing to see the huge amount of people who invest part of their time in disseminating and promoting the initiative through talks, articles, workshops, and program direction... 

Reviewer: ```suggestion
AsyncAPI and its community have grown exponentially during the last few months. As a feedback exercise, both the project and the community are getting stronger with one another. The community is the driving force that leads the initiative and the bigger it gets, the greater the consistency and potential it has. It’s amazing to see a vast number of people who invest their time in disseminating and promoting the initiative through talks, articles, workshops, and program direction... 
```

> To follow that line, the AsyncAPI Ambassador Program has been thought of as one of the latest additions to the initiative to keep the loop going. 

Reviewer: ```suggestion
To follow that line, the AsyncAPI Ambassador Program is one of the latest additions to the initiative to keep the community going. 
```

Reviewer 2: additions is plural here, fixed it in commit suggestion

> The program is, after all, a reaction to the natural impulse of the community. Thus, in this case, the ambassadors came before the Ambassadors Program itself. This proposal was born based on the recognition of the work of the community members who invest part of their time in bringing AsyncAPI to the community or new potential users. 

Reviewer: ```suggestion
The program is, after all, a reaction to the natural impulse of the community. Thus, in this case, the ambassadors came before the Ambassadors Program itself. This proposal was born based on the recognition of the work of community members who invest their time in bringing AsyncAPI to the community and new potential users. 
```

> AsyncAPI Ambassadors are passionate about open source and AsyncAPI. Committed to making AsyncAPI a place for everyone, thus participating in the leadership and mentoring of the community.

Reviewer: ```suggestion
AsyncAPI Ambassadors are passionate about open source. They are committed to making AsyncAPI an inclusive place for everyone and participate in leading and mentoring the community.
```

> The premises to be followed by anyone who wishes to become an ambassador are:

Reviewer: ```suggestion
Below are guidelines to be followed by anyone who wishes to become an ambassador:
```

- Promote user cooperation and disseminate the tools needed for any specific user.

- Engage with the project in some way, whether as a contributor, blogger, speaker, etc.

- Speak at community events or write technical content (e.g., blog posts) that solves community needs.

- Foster community growth and a welcoming environment to spread knowledge about the project.

## Why create an ambassador program?

> As we know, each person's way of looking at things is unique. Each person's abilities differ. Following that line, each ambassador can offer a different approach. They bring part of their essence into the content or the program where they are involved and disseminate. This is what brings human value enriching the conception and possibilities of the project.

Reviewer: ```suggestion
As we know, everyone has a unique way of looking at things and a person's abilities differ. Following that line, each ambassador can offer a different approach. They bring part of their essence into the content or the program where they are involved and disseminate. This is what brings human value to the project, enriching the conception and possibilities.
```

<p align="center">
  <img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOTgzODZhZDM1MWZhMDJiODNkODFhZTI3MWRjZDExZGQyYzBlNzMwYiZjdD1z/NfBApd81ooX87Od5hk/giphy.gif" alt="heart&brain" />
</p>

> We are in a time of fast expansion, which means that everything is moving very fast in the community. Last year we not only started the ambassador program, but also the AsyncAPI Mentorship program and participated in the GSoD for the first time. So, the needs of the community are changing and the initiative react being dynamic and open, driving new ideas and proposals.

Reviewer: ```suggestion
The community is experiencing rapid growth. Over the past year, we started the Ambassadors Program and the AsyncAPI Mentorship program and participated in GSoD for the first time. As the community evolves, the initiative remains dynamic and open, driving new ideas and proposals.
```

Reviewer 2: run-on sentence, fixed it

## How does the program work?

> Like any other management issue related to the initiative, the Ambassador Program is approached as a transparent system, where diversity, inclusiveness, and integration are key concepts that are wanted to be materialized in the program.

Reviewer: ```suggestion
The Ambassador Program is a transparent system, just like any other management issue related to the initiative. Its success relies on diversity, inclusiveness, and integration. The goal is to materialize these concepts within the program.
```

<p align="center">
  <img src="https://media.giphy.com/media/eEh1lwAYtnmwDWSirZ/giphy.gif" alt="heart&brain" />
</p>

> Thus, the process of recruiting ambassadors is completely open. If a person meets the requirements, they only have to add themselves to this [document](https://github.com/asyncapi/community/blob/master/AMBASSADORS_MEMBERS.json), attaching their personal information along with their contributions. The TSC members will then approve the proposal. 

Reviewer: ```suggestion
The process of recruiting ambassadors is completely open. If anyone meets the requirements, they only need to add themselves to this [document](https://github.com/asyncapi/community/blob/master/AMBASSADORS_MEMBERS.json), attaching their personal information along with their contributions. The TSC members will then approve the proposal. 
```

> Subsequently, all ambassador information is displayed on the AsyncAPI website once the PR has been merged. Like this, the process try to be as simple, collaborative, and transparent as possible.

Reviewer: ```suggestion
Subsequently, all ambassador information will be displayed on the AsyncAPI website once the PR has been merged. The process tries to be simple, collaborative, and transparent as possible.
```

## More about AsyncAPI Ambassador Program

> The minimum period of enrollment in the program is one year, with the possibility of renewal on an annual basis. At the end of each period, the number of participants will be determined and verified on a yearly basis. The minimum requirement to be an ambassador is to make four contributions per year in the form of articles, talks, videos, podcasts, presentations, drive initiatives, etc.

Reviewer: ```suggestion
The minimum enrollment period for ambassadors is one year with the option to renew annually. At the end of each period, the number of participants will be assessed and confirmed on a yearly basis. To qualify as an ambassador, individuals must make at least four contributions per year, which may include articles, talks, videos, podcasts, presentations, or driving initiatives, etc.
```

> Ambassadors acquire some smooth amount of responsibilities on holding this degree. In return, to thank the participants for their work, we try to reward their effort with some benefits, such as special swag and free conference tickets.  

Reviewer: ``` suggestion
Ambassadors are entrusted with some amount of responsibilities. As a token of appreciation for their work, we try to compensate them with certain perks, including exclusive swags and  conference passes.
```

> If you want to know more about the Ambassador Program, click [here](https://github.com/asyncapi/community/blob/6c066e0eab6c778d1a37d1beebb1418a4bc700f6/AMBASSADOR_ORGANIZATION.md).

Reviewer: ```suggestion
Learn more about the [Ambassador Program here](https://github.com/asyncapi/community/blob/6c066e0eab6c778d1a37d1beebb1418a4bc700f6/AMBASSADOR_ORGANIZATION.md).
```

Reviewer 2: fixed this commit suggestion too because it was:
- too verbose
- the link didn't adhere to accessibility standards; for anchor text links, remember to always use copy that tells a screen reader user where a link takes them (ex: **see** or **here** don't inform me anything at all)
